src/behaviours/add-cluster/__snapshots__/navigation-using-application-menu.test.tsx.snap

```diff
@@ -288,7 +288,11 @@ exports[`add-cluster - navigation using application menu when navigating to add
             </p>
             <div
               class="flex column"
-            />
+            >
+              <textarea
+                data-testid="monaco-editor-for-undefined"
```

Comment:
Some id is broken or missing? Are we sure it should end with `undefined`?

Reply:
Nah, decided not to add ID for that editor here since it would be change in implementation.